sass/overrides/_leaflet.scss

```diff
@@ -65,3 +65,10 @@ input.leaflet-control-layers-selector {
 .leaflet-map-pane svg.iconic {
     position: initial;
 }
+
+//  Override styles for reset button
+#reset-button {
+    color: black;
+    background-color: white;
+    padding: 8px 8px;
+}
```

Comment:
Could you check the styles for the small maps in edit-mode and general-view? There is an odd shadow-outline outside of the icon: 
<img width="242" alt="Screenshot 2020-02-24 at 10 46 01" src="https://user-images.githubusercontent.com/8624777/75142305-148c4480-56f3-11ea-8c7f-ed4bc2345db0.png">

Reply:
Great work getting rid of this :)